content/en/guides/hosting/iam/scim.md

```diff
@@ -327,6 +351,147 @@ DELETE /scim/Users/abc
 To temporarily deactivate the user, refer to [Deactivate user](#deactivate-user) API which uses the `PATCH` endpoint.
 {{% /alert %}}
 
+### Update User Email (Single Tennant instances only)
+
+Updates a user's primary email address. This operation is only available in self-managed single-tenant deployments.
+
+#### Endpoint
+- **URL**: `<host-url>/scim/Users/{id}`
+- **Method**: PATCH
+
+#### Parameters
+| Parameter | Type | Required | Description |
+|-----------|------|----------|-------------|
+| id | string | Yes | The unique ID of the user |
+| op | string | Yes | Must be "replace" |
+| path | string | Yes | Must be "emails" |
+| value | array | Yes | Array with new email object |
+
+#### Example
+
+{{< tabpane text=true >}}
+{{% tab header="Update Email Request" %}}
```

Comment:
I suspect these should just be "Request" and "Response" for tab headings. We are already in the section. Same feedback below.